lua/nvim-jqx/config.lua

```diff
@@ -8,8 +8,11 @@ local query_key = 'X'
 
 local sort = true
 
+local close_window_mapping = '<ESC>'
+
 return {
    geometry = geometry,
    query_key = query_key,
    sort = sort,
+   close_window_mapping = close_window_mapping
```

Comment:
probably `close_window_key` would be a more appropriate name, similar to `query_key`